### ISSUE TITLE
Access R command history rstudioapi::cmd_history(n = 25)

### ISSUE THREAD

**Issue by MilesMcBain** [rstudio/rstudioapi#50]
I'm not fully down with how history works in R but it seems like Rstudio registers a handler that causes a GUI pane to open when `history()` is called. So if I want to access the command history from R and as a character vector, there's no way to do this. 

Could the api provide this facility?

**Comment by jmcphers** [comment]
Would also be helpful for integration with `learnr`

**Comment by kmdupr33** [comment]
@jmcphers, would the team be open to a pull request that implements this?

**Comment by jmcphers** [comment]
Yup, though most of the change would need to live on the RStudio IDE side rather than in this repository. See https://github.com/rstudio/rstudio/blob/main/src/cpp/r/R/Api.R for the IDE side of many of our API methods (also note contribution instructions in https://github.com/rstudio/rstudio/blob/main/CONTRIBUTING.md).
